### ISSUE TITLE
Login Fails via webinterface

### ISSUE THREAD

**Issue by Highlander1694** [adriankumpf/teslamate#737]
Login Fails via web interface on port 3000

New install just now.  All 4 docker containers running

- database starts ok, no error
-mosquito starts ok, not errors
- grafana starts ok, no errors web browser shows grafana interface
- teslamate starts, webbrowser shows login interface, login with correct credentions results in "Error: unknown" on web interface

See teslamate console log below.

**Expected behavior**
Well it is supposed to get an API token and say something positive I suppose :)

**How to reproduce it (as minimally and precisely as possible)**:
Restarted containers, try again, same result.

**Relevant entries from the logs**

```
paste logs here
15:18:02.720 [info] Version: 1.19.3                                                                                                                                                                                              
15:18:03.231 [info] Running TeslaMateWeb.Endpoint with cowboy 2.7.0 at :::4000 (http)                                                                                                                                            
15:18:03.295 [info] Access TeslaMateWeb.Endpoint at http://localhost                                                                                                                                                             
15:18:03.457 [info] MQTT connection has been established                                                                                                                                                                         
15:19:38.635 [info] GET /                                                                                                                                                                                                        
15:19:38.762 [info] Sent 302 in 126ms                                                                                                                                                                                            
15:19:58.814 [info] GET /                                                                                                                                                                                                        
15:19:58.824 [info] Sent 302 in 9ms                                                                                                                                                                                              
15:19:58.931 [info] GET /sign_in                                                                                                                                                                                                 
15:19:58.988 [info] Sent 200 in 56ms                                                                                                                                                                                             
15:20:29.259 [info] Elixir.TeslaMate.Api / unhandled message: {#Reference<0.2590530517.636485633.34428>,                                                                                                                         
 {:error, %Mint.TransportError{reason: :nxdomain}}}                                                                                                                                                                              
15:20:38.397 [info] GET /                                                                                                                                                                                                        
15:20:38.448 [info] Sent 302 in 50ms                                                                                                                                                                                             
15:23:13.241 [warn] Update check failed: %Mojito.Error{message: nil, reason: :checkout_timeout}                                                                                                                                  
15:23:13.276 [error] GenServer TeslaMate.Updater terminating                                                                                                                                                                     
** (FunctionClauseError) no function clause matching in TeslaMate.Updater.handle_info/2                                                                                                                                          
    (teslamate 1.19.3) lib/teslamate/updater.ex:69: TeslaMate.Updater.handle_info({#Reference<0.2590530517.636485633.34691>, {:error, %Mint.TransportError{reason: :nxdomain}}}, %TeslaMate.Updater.State{update: nil, version: "
1.19.3"})                                                                                                                                                                                                                        
    (stdlib 3.12.1) gen_server.erl:637: :gen_server.try_dispatch/4                                                                                                                                                               
    (stdlib 3.12.1) gen_server.erl:711: :gen_server.handle_msg/6                                                                                                                                                                 
    (stdlib 3.12.1) proc_lib.erl:249: :proc_lib.init_p_do_apply/3                                                                                                                                                                
Last message: {#Reference<0.2590530517.636485633.34691>, {:error, %Mint.TransportError{reason: :nxdomain}}}                                                                                                                      
15:23:25.558 [info] Elixir.TeslaMate.Api / unhandled message: {#Reference<0.2590530517.636485634.34278>,                                                                                                                         
 {:error, %Mint.TransportError{reason: :nxdomain}}}                                                                                                                                                                              
15:23:56.811 [info] GET /                                                                                                                                                                                                        
15:23:56.944 [info] Sent 302 in 132ms                                                                                                                                                                                            
15:24:53.227 [info] GET /                                                                                                                                                                                                        
15:24:53.380 [info] Sent 302 in 152ms                                                                                                                                                                                            
15:25:03.318 [info] GET /                                                                                                                                                                                                        
15:25:03.500 [info] Sent 302 in 181ms 
```

**Screenshots**

<!-- If applicable, add screenshots to help explain the problem. -->

**Data**



**Environment**

- TeslaMate version: log says: 1.19.3 (just installed 21.05.2020)
- Type of installation: Docker 
- OS TeslaMate is installed on: Linux
- User OS & Browser: Firefox tried on Edge too
- Others:

**Comment by Highlander1694** [comment]
Found out that the docker is not resolving DNS.  Login on the docker container shows DNS resolution finds no valid servers.

Problem seems to be related to this: https://github.com/pi-hole/docker-pi-hole/issues/455

**Comment by Highlander1694** [comment]
Found the solution.  QNAP Docker (for some reason) makes a /etc/resolv.conf with a DNS resolver of 127.0.0.11.  That should be 127.0.0.1, not 11...

Solution.  Delete the Docker containers.  Redo the containers with the change below.

Add the following lines before the "environment:" and make sure the "cap_add" and "dns:" are aligned with the "environment:" (same distance from left margin)
    cap_add:
      - NET_ADMIN
    dns:
      - 127.0.0.1
      - 1.1.1.1

Further Reading:
I am a Docker newb, I found the hint for the solution here https://github.com/pi-hole/docker-pi-hole/issues/410
